indigo/indigo/src/main/scala/indigo/platform/events/WorldEvents.scala

```diff
@@ -23,6 +23,11 @@ import org.scalajs.dom.document
 import org.scalajs.dom.html
 import org.scalajs.dom.window
 
+import scala.scalajs.js.Date
+
+@SuppressWarnings(Array("scalafix:DisableSyntax.var"))
+var pointerButtons: Map[Double, Batch[(Int, Date)]] = Map.empty
```

Comment:
This can't be here as a top level this, needs to move inside the class at the very least, i.e.:

```scala
final class WorldEvents:
  @SuppressWarnings(Array("scalafix:DisableSyntax.var"))
  var pointerButtons: Map[Double, Batch[(Int, Date)]] = Map.empty
```

Reply:
And it should probably be `private`.

Reply:
All done